esmvaltool/cmor/tables/cmip5/Tables/CMIP5_Omon

```diff
@@ -184,7 +184,7 @@ long_name:        ocean passage
 !----------------------------------
 ! Additional axis information:
 !----------------------------------
-out_name:         line
+out_name:         passage
```

Comment:
Please file an issue with the source of the CMOR tables:
https://github.com/PCMDI/cmip5-cmor-tables/issues

Our tables are a copy of those, so your changes in the esmvaltool repo will be lost if we ever update.

Reply:
Created an issue with CMOR [here](https://github.com/PCMDI/cmip5-cmor-tables/issues/1) (It's issue number 1!)